12.0 july2022

stephansainleger opened the pull request.

Commit:
[UPD] Add membership state info on portal home page (fixes #241)

Commit:
[UPD] Update translations

Commit:
[IMP] lcc_members: add wallet creation refusal field

Fixes #242

Commit:
[IMP] lcc_lokavaluto_app_con..: add wallet creation in private form

Adds field so the connected user can refuse the creation of a numeric
wallet from the private registration form.

Fixes #242

Commit:
[IMP] lcc_lokavaluto_app_con..: add wallet creation in orga form

Adds field so the connected user can refuse the creation of a numeric
wallet from the organization registration form.

Fixes #242

Commit:
[IMP] lcc_lokavaluto_app_connection: add french translations

Commit:
[IMP] lcc_cyclos_base: add french translations

Commit:
[IMP] lcc_comchain_base: add french translations

Commit:
[REM] lcc_members_website: remove old addon

Commit:
[REM] lcc_partner_gogocarto_export_api: remove old addon

Commit:
[IMP] lcc_members: re-organize partner form fields to ease reading

Commit:
[IMP] lokavaluto_app_con..: add commercial name in recipient search

adds the commercial name in the recipient research criteria

Fixes #244

Commit:
[IMP] lcc_lokavaluto_app_con..: add wallet creation in web orga form

njeudy commented on lcc_lokavaluto_app_connection/services/partner_services.py:
```diff
@@ -146,7 +146,9 @@ def search_recipients(self, recipients_search_info):
                     "|",
+                    "|",
                     ("partner_id.public_profile_id.name", "ilike", value),
+                    ("partner_id.public_profile_id.company_name", "ilike", value),
```
Ce n'est pas le bon champ, company_name est utilisé pour saisir juste une société sur une fiche contact si on ne veux pas créer la fiche société .. ce n'est pas le nom commercial .. il faut créer un champ je pense (je crois qu'on a une issue la dessus)

stephansainleger replied:
nouveau champs `business_name` créé

njeudy commented on lcc_lokavaluto_app_connection/services/partner_services.py:
```diff
@@ -312,6 +314,7 @@ def _get_partner_parser(self):
         parser = [
             "id",
             "name",
+            "company_name",
```
Idem pas le bon champ, ce n'est pas le nom commercial dans odoo

stephansainleger replied:
nouveau champs `business_name` créé

Commit:
[IMP] lcc_members: add business name field to partner model

fixes #244

Commit:
[IMP] lcc_members_portal: change business name in registering form

fixes #244

Commit:
[IMP] lcc_lokavaluto_app_connection: fix search on business name

fixes #244

Commit:
[REM] lcc_members_portal: remove old code

Commit:
[FIX] lcc_members_portal: correct local group retrieval in forms

Commit:
[IMP] lcc_members_portal: move the registery request type in tags

Moves the "NEW APPLICATION", "AFFILIATION" and "RENEWAL" data from the
lead title to the lead tags.

Fixes #245

Commit:
[IMP] lcc_members_portal: add filters on opportunities

add filters to retrieve application, affiliation or renewal
opportunities

Fixes #213

Commit:
[IMP] lcc_members_portal: update french translations

Commit:
[IMP] all: update missing french translations
